### PR TITLE
KeyboardManager Addition

### PR DESCRIPTION
**Pull request by nathantannar4**
Allow the MessageInputBar to be used as a subview rather than an inputAccessoryView

### CODE REVIEW

**Comment by SD10** [comment]
@nathantannar4 Is this the full PR? 🤔

File: Sources/Models/KeyboardNotification.swift [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+ AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ SOFTWARE.
+ */
+
+import Foundation
+
+public struct KeyboardNotification {
+
```
Comment (SD10) [inline Sources/Models/KeyboardNotification.swift]: I think this type is not `MessageInputBar` specific enough to be in this project. Ideally, the `MessageInputBar` should have no idea that it is the `inputAccessoryView`. A user can use it as such or as any other view. But I want to ignore the notion of a keyboard

**Review by SD10 — COMMENTED** [review]
I actually have another project in progress, solely related to keyboard handling. Let's re-evaluate after I get the current features like autocomplete manager in MessageKit

**Comment by nathantannar4** [comment]
@SD10 file system only had the reference to the file, KeyboardManager there now

**Pull request closed without merging**